tests/php/test-tag-and-attribute-sanitizer.php

```diff
@@ -1256,27 +1256,23 @@ static function() {
 				[ AMP_Tag_And_Attribute_Sanitizer::INVALID_URL_PROTOCOL ],
 			],
 
-			'a_with_wrong_host'                            => [
+			'a_with_space_in_url'                          => [
 				'<a class="foo" href="http://foo bar">value</a>',
-				'<a class="foo">value</a>',
-				[],
-				[ AMP_Tag_And_Attribute_Sanitizer::INVALID_URL ],
+				'<a class="foo" href="http://foo%20bar">value</a>',
 			],
 			'a_with_encoded_host'                          => [
 				'<a class="foo" href="http://%65%78%61%6d%70%6c%65%2e%63%6f%6d/foo/">value</a>',
 				null,
 			],
-			'a_with_wrong_schemeless_host'                 => [
-				'<a class="foo" href="//bad domain with a space.com/foo">value</a>',
-				'<a class="foo">value</a>',
-				[],
-				[ AMP_Tag_And_Attribute_Sanitizer::INVALID_URL ],
+			'a_with_spaces_inside'                         => [
+				'<a class="foo" href="//domain with space.com/foo">value</a>',
+				'<a class="foo" href="//domain%20with%20space.com/foo">value</a>',
+			],
+			'a_with_schemaless_host'                       => [
+				'<a class="foo" href="//domain.om/foo">value</a>',
 			],
 			'a_with_mail_host'                             => [
-				'<a class="foo" href="mail to:foo@bar.com">value</a>',
```

Comment:
I think the space in:

```html
mail to:foo@bar.com
```
...is what caused this be invalid. 

Passing `<a class="foo" href="mailto:foo@bar.com">` to the [AMP validator](https://validator.ampproject.org/) shows that it's valid.

Reply:
This should be resolved by implementing my change here: https://github.com/ampproject/amp-wp/pull/4166#discussion_r370468687

The `mail to`  protocol should still be invalid, so this chunk should be reverted. 

This code: `<a class="foo" href="mail to:foo@bar.com">` should be invalid.

Reply:
Sure, good point. That's applied in https://github.com/ampproject/amp-wp/pull/4166/commits/f19301917f553dd7cc66d593d2fa0e6a02160e49